_gsocproposals/2023/proposal_E2E4.md

```diff
@@ -1,39 +1,31 @@
 ---
-title: Vision Transformers for End-to-End Particle Reconstruction for the CMS Experiment
+title: Equivariant Neural Networks to explore the underlying symmetries in particle physics events
```

Comment:
this should be a new proposal, not replacing the Vision transformers project. Please revert the changes to E2E4 (except for the mentors update) and create a new proposal E2E5 with the Equivariant networks